src/Orchard.Web/Modules/Orchard.Alias/Implementation/Map/AliasMap.cs

```diff
@@ -171,7 +171,8 @@ public class AliasMap {
             }
 
             if (match == null) {
-                var foundPath = focus.Paths.Keys.FirstOrDefault();
+                // Get the shortest path available to ensure the "home" alias is always taken if present.
+                var foundPath = focus.Paths.Keys.OrderBy(x => x).FirstOrDefault();

```

Comment:
I feel like OrderBy Key isent the right thing. what if my alias a length of 1 and wasent / ?